panther_analysis_tool/main.py

```diff
@@ -223,9 +223,17 @@ def zip_analysis(args: argparse.Namespace) -> Tuple[int, str]:
     current_time = datetime.now().isoformat(timespec='seconds').replace(
         ':', '-')
     filename = 'panther-analysis'
-    return 0, shutil.make_archive(
-        os.path.join(args.out, '{}-{}'.format(filename, current_time)), 'zip',
-        args.path)
+    zip_out = zipfile.ZipFile('{}-{}.zip'.format(filename, current_time), 'w',
```

Comment:
ZipFile is a context manager, so instead of having to `.close()` it explicitly, you can say:

```python
with zipfile.ZipFile(...) as zip_out:
    zip_out.write(...)
```

This will also close the file if there were any exceptions

Reply:
Good call!